Formula/m/mavsdk.rb

```diff
@@ -77,35 +64,27 @@ def install
 
     ENV.llvm_clang if OS.mac? && (DevelopmentTools.clang_build_version <= 1100)
 
-    # Install protoc-gen-mavsdk deps
-    venv_dir = buildpath/"bootstrap"
-    venv = virtualenv_create(venv_dir, "python3.12")
-    venv.pip_install resources
-
-    # Install protoc-gen-mavsdk
-    venv.pip_install "proto/pb_plugins"
-
-    # Run generator script in an emulated virtual env.
-    with_env(
-      VIRTUAL_ENV: venv_dir,
-      PATH:        "#{venv_dir}/bin:#{ENV["PATH"]}",
-    ) do
-      system "tools/generate_from_protos.sh"
```

Comment:
As note, upstream now automatically sets up Python build environment (https://github.com/mavlink/MAVSDK/blob/v2.3.0/tools/generate_from_protos.sh#L93-L94) so creating venv and the resources don't seem to do anything.

For now, I removed all the re-generation logic. If we still hit issues on future `protobuf` bump, then we can try restoring running `tools/generate_from_protos.sh`.